webapp/channels/.eslintignore

```diff
@@ -1,3 +1,4 @@
 node_modules
 dist
 lib
+src/i18n/imports.ts
```

Comment:
Instead of ignoring this from here, I think it'd be better to add a `/* eslint-disable */` comment to the top of `imports.ts`. That'll only affect that one file